extensions/amp-story/1.0/page-advancement.js

```diff
@@ -475,6 +475,13 @@ export class ManualAdvancement extends AdvancementConfig {
           return true;
         }
 
+        if (
+          el.classList.contains('i-amphtml-story-screen-reader-back-button')
+        ) {
+          shouldHandleEvent = true;
+          return true;
+        }
```

Comment:
Insider tip: you don't need this block if you add a `role="button"` to your button

Reply:
This is awesome and will be helpful when building other UI. 

We need this here though because we want the event to trigger a page navigation.